sphinx/source/governance.rst

```diff
@@ -44,34 +44,50 @@ As soon as ``member3`` accepts the proposal, a quorum (2 out of 3) of members ha
 
     .. code-block:: bash
 
-        $ ../build/memberclient ack --cert=member4_cert.pem --privk=member4_privk.pem --host=10.1.0.4 --port=25000 --ca=networkcert.pem --sign
+        $ memberclient --server-address 127.83.203.69:55526 --cert member4_cert.pem --privk member4_privk.pem --ca networkcert.pem ack
         {"commit":108,"global_commit":107,"id":2,"jsonrpc":"2.0","result":true,"term":2}
 
 
 Displaying proposals
 --------------------
 
-The details of pending proposals, including the proposer member ID, proposal script, parameters and votes, can be displayed with the ``proposal_display`` option of the ``memberclient`` utility. For example:
+The details of pending proposals, including the proposer member ID, proposal script, parameters and votes, can be displayed with the ``proposal_display`` sub command of the ``memberclient`` utility. For example:
 
 .. code-block:: bash
 
-    $ memberclient proposal_display --cert=member1_cert.pem --privk=member1_privk.pem --host=10.1.0.4 --port=25000 --ca=networkcert.pem
-    ------ Proposal ------
-    -- Proposal id: 1
-    -- Proposer id: 0
-    -- Script: {"text":"\n      tables, member_cert = ...\n      return Calls:call(\"new_member\", member_cert)\n    "}
-    -- Parameter: [<member_cert>]
-    -- Votes: [[1,{"text":"\n      tables, changes = ...\n      return false"}]]
-    ----------------------
-
-In this case, there is one pending proposal (``id`` is 1), proposed by the first member (``member1``, ``id`` is 0) and which will call the ``new_member`` function with the new member's certificate as a parameter. Only one vote has been cast by ``member2`` (``id`` is 1) to reject the proposal while ``member1`` (proposer) has already implicitly accepted it.
+    $ memberclient --server-address 127.83.203.69:55526 --cert=member1_cert.pem --privk=member1_privk.pem --ca=networkcert.pem proposal_display
```

Comment:
Should we stick to `--option=` (instead of `--option `) for this doc?

Reply:
Have opted for consistency on `--option value`.